package.json

```diff
@@ -158,12 +153,11 @@
     "mochawesome-report-generator": "^6.2.0",
     "playwright-ctrf-json-reporter": "^0.0.18",
     "rollup-plugin-polyfill-node": "^0.11.0",
-    "typescript": "^5.0.4",
+    "storybook": "9.0.18",
+    "typescript": "5.8.3",
```

Comment:
Note: pinned version of TypeScript due to using `^` (probably a mistake) we were already using latest version 😄

Reply:
Yes, this comment was incorrect, I forgot to remove it 😅